pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -35,6 +35,14 @@
     // ZooKeeper session timeout
     private int zookeeperSessionTimeoutMs = 30_000;
 
+    // If Discovery Service is Disabled the proxy will just authenticate the client 
+    // and forward all requests to a VIP or any other service discovery port 
+    private boolean discoveryServiceEnabled = true;
+    
+    // if Service Discovery is Disabled this url should point to the discovery service provider. 
+    private String discoveryServiceURL = "pulsar://localhost:6650/";
```

Comment:
should we rename it to `brokerServiceUrl` as in this case proxy will directly try to communicate broker with brokerserviceUrl?